tests/TestCase/Http/ClientTest.php

```diff
@@ -59,6 +59,12 @@ public function testConstructConfig()
         }
     }
 
+    public function testConstructWithEmptyOptions()
+    {
+        $http = new Client();
+        $this->assertEquals('Cake\Http\Client\Adapter\Stream', $http->getConfig('adapter');
```

Comment:
You don't want to use ::class syntax?

Reply:
A closing brace `)` is missing, causing parse error.